packager/packager.cc

```diff
@@ -231,6 +232,95 @@ ast::UnresolvedConstantLit *verifyConstant(core::MutableContext ctx, core::NameR
     return target;
 }
 
+struct EnforcePackagePrefix {
```

Comment:
You can make this a class; the traversal methods and constructor just need to be in a `public:` section.